src/modules/verifiable-credentials/storage-client.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  PresentationDefinition,
+  VerifiableCredential,
+  CredentialSubject,
+} from '@ew-did-registry/credentials-interface';
+import { AxiosInstance } from 'axios';
+import { CacheClient } from '../cacheClient';
+
+export default class VCStorageClient {
+  private _http: AxiosInstance;
+
+  constructor(cacheClient: CacheClient) {
+    this._http = cacheClient.http;
+  }
+
+  async getCredentialsByDefinition(
+    definition: PresentationDefinition
+  ): Promise<VerifiableCredential<CredentialSubject>[]> {
+    const { data } = await this._http.post<
+      VerifiableCredential<CredentialSubject>[]
+    >(`/match`, { baseURL: 'vp', params: definition });
```

Comment:
```suggestion
    >(`/vp/match`, definition);
```
We can't override `baseUrl` here, bc it's actually overriding SSI-Hub URL. Also `definition` should be in body not in params.